tf2onnx/utils.py

```diff
@@ -437,3 +448,52 @@ def is_onnx_domain(domain):
     if domain is None or domain == "":
         return True
     return False
+
+
+def build_onnx_op(node):
+    """Build onnx op"""
+    onnx_node = helper.make_node(node.type, node.input, node.output, name=node.name)
+    # deal with attributes
+    attr = []
+    attr_graphs = node.get_body_graphs()
+    if attr_graphs:
```

Comment:
i am curious about whether we can infer node with sub graph correctly.
you may want to add tests, e.g. a simple If/Loop/Scan?